addons/l10n_ec/data/res_bank_data.xml

```diff
@@ -0,0 +1,111 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@Stentor duplicated data, it is better to have the csv.

Reply:
Solved in incoming commit